modules/nf-core/oncocnv/main.nf

```diff
@@ -0,0 +1,73 @@
+process ONCOCNV {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::oncocnv=7.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://containers.biocontainers.pro/s3/SingImgsRepo/oncocnv/v7.0_cv2/oncocnv_v7.0_cv2.sif':
+        'registry.hub.docker.com/biocontainers/oncocnv:v7.0_cv2' }"
+
+    input:
+    tuple val(meta), path(normal), path(normal_index), path(tumor), path(tumor_index)
+    path bed
+    path fasta
+
+    output:
+    tuple val(meta), path("*.profile.png")  ,emit: png
+    tuple val(meta), path("*.profile.txt")  ,emit: profile
+    tuple val(meta), path("*.summary.txt")  ,emit: summary
+    path "versions.yml"                     ,emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def cghseg = task.ext.args2 ?: 'cghseg'
+    def mode = task.ext.args ?: '-m Ampli'
+    def normal = normal.join(',')
+    def tumor = tumor.join(',')
+    def VERSION = '7.0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    """
+    perl \$(which ONCOCNV_getCounts.pl) \\
+        getControlStats \\
+        $mode \\
+        -b ${bed} \\
+        -c $normal \\
+        -o ControlStats.txt
+
+    perl \$(which ONCOCNV_getCounts.pl) \\
+        getSampleStats \\
+        $mode \\
+        -c ControlStats.txt \\
+        -s $tumor \\
+        -o SampleStats.txt
+
```

Comment:
how big long running are these jobs? wondering of separate modules would make more sense...

Reply:
considering that they process several samples these jobs are very short. For the samples we worked with the whole thing finished in under 15 min. As an additional argument to keep them combined I would like to mention that the ONCOCNV repository contains a bash script to run these processes combined but the script needed. I omitted using the script itself as it is difficult to work with - sample input etc. needs to be set manually in the script.

Reply:
ok, still feels a bit weird for me. But also don't have a good counter idea at the moment